src/driver/op.rs

```diff
@@ -40,14 +42,18 @@ pub(crate) struct Op<T: 'static, CqeType = SingleCQE> {
 /// A Marker for Ops which expect only a single completion event
 pub(crate) struct SingleCQE;
 
+/// A Marker for Ops which may be submitted when the ring is not present
+/// The only user of this is `Close` currently
+pub(crate) struct Fallible;
```

Comment:
I'm unconvinced this is needed. changin t5ry_submit_with to try_submit_with_immediate or similar, which would submit the operation immediately would be sufficient